src/support/index.coffee

```diff
@@ -105,7 +105,10 @@ module.exports = ->
     argv['prevent-browser-reload']?
 
   terminateDriver = =>
-    @driver.close()
-    @driver.quit()
+    if @driver?
+      @driver.close()
+      @driver.quit()
+    else
+      $.defer()
```

Comment:
Yeh man I'll sort it tomorrow. Hate depending on our own forks!



Cheers,Tom

On Mon, Apr 27, 2015 at 10:40 PM, Sam Saccone <notifications@github.com>
wrote:

>> @@ -105,7 +105,10 @@ module.exports = ->
>>      argv['prevent-browser-reload']?
>>  
>>    terminateDriver = =>
>> -    @driver.close()
>> -    @driver.quit()
>> +    if @driver?
>> +      @driver.close()
>> +      @driver.quit()
>> +    else
>> +      $.defer()
> yeah I think this the idea behind this is a good PR. mind just updating a touch and we can get this merged in?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/mojotech/pioneer/pull/319/files#r29194433